GillianCore/parserAndCompiler/ParserAndCompiler.mli

```diff
@@ -8,6 +10,12 @@ type ('annot, 'tl_ast, 'init_data) compiled_progs = {
   init_data : 'init_data;
 }
 
+val get_progs_or_fail :
+  (Format.formatter -> 'e -> unit) ->
+  (('a, 'b, 'c) compiled_progs, 'e) result ->
+  ('a, 'b, 'c) compiled_progs
```

Comment:
Some documentation on this function
Also, the first argument is quite surprising, I would make it labeled, `~pp_err`
My rule of thumb for making arguments labeled is: do I already know what the argument is given the function name and the argument type?

```ocaml
(* Doesn't need labels *)
val use_query: database -> query -> query_result

(* Wait, what's this weird first argument? Better make it labeled *)
val use_query: (string -> string) -> database -> query -> query_result

(* Arguments have the same type, how do I know which one is which? *)
val copy_content: channel -> channel -> unit

(* This is better *)
val copy_content : in:channel -> out:channel -> unit
```

Reply:
Very good point, well made :)
So well made, in fact, that I'm going to add that to the style guide!

Made the change in 3d2cc07.